src/messages/variant.rs

```diff
@@ -30,8 +30,10 @@ use xor_name::XorName;
 pub(crate) enum Variant {
     /// Inform neighbours about our new section.
     NeighbourInfo {
-        /// `EldersInfo` of the neighbour section.
-        elders_info: EldersInfo,
+        /// `EldersInfo` of the sender's section, with the proof chain.
+        elders_info: Proven<EldersInfo>,
+        /// Latest section key of the sender's section
+        section_key: bls::PublicKey,
```

Comment:
This might not be needed because we already have the key - it's the latest key of the proof chain of this message.